### ISSUE TITLE
Project

### ISSUE THREAD

**Issue by ricardoasmartins** [tensorflow/models#9964]
I'm doing a project to detect people from a top view.  I'm using a raspberry pi 4 and Google coral to run the model.  Right now I am using the mobile det model trained in the COCO dataset and I have good results but I would like to improve the model to detect people in some specific cases as a person.  Should I do fine tuning?  if so how?

**Comment by ricardoasmartins** [comment]
How do I fine-tune only the person class ?

**Comment by bundelesneha05** [comment]
Only allow the labels/groundtooth of the person while creating the tfrecords. Plz, check the create_coco_tf_record.py. And check some augmentation techniques for top view.

**Comment by saikumarchalla** [comment]
@ricardoasmartins Please go through this SO [link](https://stackoverflow.com/questions/56012092/tensorflow-object-detection-api-how-does-the-fine-tuning-of-a-model-works) for fine tuning OD model. Also, as mentioned above you can use data augmentation techniques to improve the performance of the model.
